index.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+const request = require('request-promise');
+
+/**
+ * @param {{name: string, version: string}} packageData
+ */
+module.exports = function getRequestWrapper(packageData) {
```

Comment:
I think we should pass `packageData` as well as a reference to `request-promise`, maybe not bother making it a dependency?